client/src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -25,34 +33,57 @@ class Breadcrumb extends Component {
     ));
   }
 
+  /**
+   * @returns {*}
+   */
   renderLastCrumb() {
     const crumb = this.getLastCrumb();
     if (!crumb) {
       return null;
     }
-
-    const iconClassNames = ['breadcrumb__icon'];
-    if (crumb.icon) {
-      iconClassNames.push(crumb.icon.className);
-    }
-
     return (
       <div className="breadcrumb__item breadcrumb__item--last">
         <h2 className="breadcrumb__item-title">
           {crumb.text}
-          {crumb.icon && (
-            <span
-              className={iconClassNames.join(' ')}
-              role="button"
-              tabIndex={0}
-              onClick={crumb.icon.onClick}
-            />
-          )}
+          {crumb.icon && this.renderIcons([crumb.icon])}
+          {crumb.icons && this.renderIcons(crumb.icons)}
         </h2>
       </div>
     );
   }
 
+  /**
+   * @param {Array} icon
+   * @returns {*}
+   */
+  renderIcons(icons) {
+    return icons.map(icon => {
+      const { nodeName, className, hasRestrictedAccess, ...other } = icon;
+      // reassign with let so linter won't suggest 'const' above for unmodified nodeName/className
+      let attrs = { ...other };
+      const classNames = ['breadcrumb__icon'];
+      if (className) {
+        classNames.push(className);
+      }
+      attrs = { tabIndex: '0', ...attrs };
+      if (attrs.hasOwnProperty('onClick')) {
+        attrs = { role: 'button', ...attrs };
+      }
```

Comment:
Any element can have a click event attached to it, but only buttons need `role=button`.

```suggestion
      if (nodeName === 'button') {
        attrs = { role: 'button', ...attrs };
      }
```

Reply:
True, though there's some legacy code that passes in an object with an 'onClick' though not a 'nodeName'.  I'd rather not refactor legacy code in a feature pr if I can avoid it.

Reply:
I'm a doofus, I was thinking of `type="button"` 🙃